content/may2015/openid-connect.md

```diff
@@ -0,0 +1,153 @@
+    "refresh_token": "daf989e2-649a-48c8-b2d8-18efa70be0ef",
+    "scope": "user/Patient.read online_access openid",
+    "id_token": "eyJraWQiOiIyMDE2LTEyLTE0VDE2OjQ5OjIxLjUxMS5yc2EiLCJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiJwb3J0YWwiLCJhdWQiOiI5NGJiZDkwZC00ODJhLTRhMTAtYjdkZi1iNDBlZGIyNzhkYTIiLCJpc3MiOiJodHRwczpcL1wvYXV0aG9yaXphdGlvbi5zYW5kYm94Y2VybmVyLmNvbVwvdGVuYW50c1wvMGI4YTAxMTEtZThlNi00YzI2LWE5MWMtNTA2OWNiYzZiMWNhXC9vaWRjXC9pZHNwc1wvMGI4YTAxMTEtZThlNi00YzI2LWE5MWMtNTA2OWNiYzZiMWNhXC8iLCJleHAiOjE0ODE3NDc3ODQsImlhdCI6MTQ4MTc0NzE4NH0.YvF8Pqelk6Ze4srAjUnHZ192-bxaglYcbvbf8_AfDXtsmLKcSfHtQ9MmeGX5k6E-Mv4sqDEwzKPmU8p2GifOzYeJwu-TvJDCMDoqMk7WSLbb9wJk2gJ3Ipi7SEQcyVdPBGx9GMb4oX00bom0aliT2v-oGUeCcGOIPjVFM3eVaxgaHo7L_4shRX-FbUa96ORarD0c92m4PZZoiHqujCuRC7TjRfIvPXnBCCCl5naEo-gqkTNEfSf_jpfZPLAsQgW5jFcFiz9ihrBgM6A7yIE-1wSpIm9EiwY5gm3lJQTyGzYg3tzoOHIBWzbVTeTYTUlNMiwC_icHv2j9FDWHSxKTfw",
+    "token_type": "Bearer",
+    "expires_in": 570
+}
+</pre>
+
+The claims of the identity token in the above example are as follows:
+
+<pre class="terminal">
+{
+  "sub": "portal",
+  "aud": "94bbd90d-482a-4a10-b7df-b40edb278da2",
+  "iss": "https://authorization.sandboxcerner.com/tenants/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/oidc/idsps/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/",
+  "exp": 1481747784,
+  "iat": 1481747184
+}
+</pre>
+
+In this example, the user identifier is "portal", the issuer is
+``https://authorization.sandboxcerner.com/tenants/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/oidc/idsps/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/``
+and the recipient client id is ``94bbd90d-482a-4a10-b7df-b40edb278da2``.
+
+The following is an example URL for the location of an OpenID Connect
+configuration document, derived from the issuer in the above id_token:
+
+``https://authorization.sandboxcerner.com/tenants/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/oidc/idsps/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/.well-known/openid-configuration``
+
+The following is example content of the configuration document:
+
+<pre class="terminal">
+{
+    "response_types_supported": ["code"],
+    "request_parameter_supported": false,
+    "require_request_uri_registration": false,
+    "request_uri_parameter_supported": true,
+    "claims_parameter_supported": false,
+    "jwks_uri": "https:\/\/authorization.sandboxcerner.com\/jwk",
```

Comment:
The example came from the actual responses from the server - probably best the examples are in-line with actual output.